.github/workflows/ci.yml

```
@@ -0,0 +1,77 @@
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      # Setup Bun with specific version
+      - name: Setup Bun
+        uses: oven-sh/setup-bun@v1
+        with:
+          bun-version: 1.0.14
+
+      # Cache Bun dependencies
+      - name: Cache Bun dependencies
+        uses: actions/cache@v3
```

actionlint (1.7.4)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)